main.tf

```diff
@@ -64,7 +64,7 @@
 
 module "dcos-core" {
   source  = "dcos-terraform/dcos-core/template"
-  version = "~> 0.2.11"
+  version = "~> 0.2.12"
```

Comment:
`dcos-terraform/dcos-core/template` will cut a new release to include calico configurations